cluster/pulumi/deployment/src/config.ts

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { config, DeploySvRunbook } from '@lfdecentralizedtrust/splice-pulumi-common';
+import {
+  mustInstallSplitwell,
+  mustInstallValidator1,
+} from '@lfdecentralizedtrust/splice-pulumi-common-validator';
+import { clusterYamlConfig } from '@lfdecentralizedtrust/splice-pulumi-common/src/config/configLoader';
+import { z } from 'zod';
+
+const ProjectFilterSchema = z.union([
+  z.literal('canton-network'),
+  z.literal('gha'),
+  z.literal('infra'),
+  z.literal('multi-validator'),
+  z.literal('splitwell'),
+  z.literal('sv-canton'),
+  z.literal('sv-runbook'),
+  z.literal('validator-runbook'),
+  z.literal('validator1'),
+]);
+
+export type ProjectFilter = z.infer<typeof ProjectFilterSchema>;
+
+const defaultProjectFilters = [
+  'canton-network',
+  'infra',
+  'sv-canton',
+  'validator-runbook',
+] as const;
+
+const DeploymentConfigSchema = z.object({
```

Comment:
we should probably merge within the same file as OperatorDeploymentConfigSchema

I agree it's a bit awkward the location because of reasons but better than having the config split